lib/credit_card_validations/plugins/voyager.rb

```diff
@@ -0,0 +1 @@
+CreditCardValidations.add_brand(:voyager, {length: 15, prefixes: '86'})
```

Comment:
Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Final newline missing.